amistuck.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# This script is intended for use as a liveness probe of an ActiveMQ server.
+# It expects the name of a queue to monitor and an optional threshold value
+# in seconds (defaults to 300). The script uses the Jolokia API endpoint to
+# find the current size of the queue. If that size differs from a previously
+# recorded size, or there is no previously recorded size (such as during the
+# first # execution of the script), it simply records the current size of
+# the queue and exits normally.
+#
+# If, however, the current size is the same as the previously recorded size
+# (and it is not 0), then the script compares the current time to the last
+# time the size was recorded as changing. If that value is greater than the
+# threshold, the script emits a status message to STDOUT and exits with a
+# nonzero status.
+
```

Comment:
Universally a good thing to add to bash scripts:

```suggestion
set -euo pipefail
```